src/lib/tabs/tab-nav-bar/tab-nav-bar.ts

```diff
@@ -40,10 +52,26 @@ export class MdTabNavBar {
   /** Checks if the active link has been changed and, if so, will update the ink bar. */
   ngAfterContentChecked(): void {
     if (this._activeLinkChanged) {
-      this._inkBar.alignToElement(this._activeLinkElement.nativeElement);
+      this._alignInkBar();
       this._activeLinkChanged = false;
     }
   }
+
+  ngOnDestroy() {
+    if (this._directionChange) {
+      this._directionChange.unsubscribe();
+      this._directionChange = null;
+    }
+  }
+
+  /** Aligns the ink bar to the active link. */
+  private _alignInkBar(): void {
+    this._ngZone.runOutsideAngular(() => {
```

Comment:
Would it make sense to move the `runOutsideangular` and `requestAnimationFrame` part of this from both tab-group and tab-nav-bar into the ink-bar?